.github/workflows/reusable-jira-workflow.yml

```diff
@@ -22,12 +22,12 @@ jobs:
             }
 
             Set-JiraConfigServer -Server "${{ secrets.JIRA_URL }}"
-            if("${{ inputs.branch_name }}" -match "[a-zA-Z]+[_-][0-9]+" -eq $False)
+            if("${{ inputs.branch_name }}" -match "\/[a-zA-Z]+[_-][0-9]+" -eq $False)
             {
               throw "Branch name ${{ inputs.branch_name }} doesn't respect the required pattern ([a-zA-Z]+[_-][0-9]+). A valid branch name example would be: feature/IDP-123"
             }
 
-            $JiraIssueKey = $Matches[0]
+            $JiraIssueKey = $Matches[0] -replace '/', ''
```

Comment:
oh actually, can we also replace the `_` by `-` if there is any? Currently we allow `IDP-123` and also `IDP_123` but turns out the Jira api does not allow underscores as seen here: https://github.com/gsoft-inc/wl-domain-event-propagation/actions/runs/8285877798

Reply:
I think Jira allows for underscore in card Id, it's just that in your case IDP_123 doesn't exist in Jira because it's IDP-123 but that doesn't mean another team wouldn't have an identifier with an underscore instead of a dash